arangod/Aql/OptimizerUtils.cpp

```diff
@@ -443,7 +443,9 @@ std::pair<bool, bool> findIndexHandleForAndNode(
 
   if (bestIndex == nullptr) {
     for (auto const& idx : indexes) {
-      considerIndex(idx);
+      if (!Index::onlyHintForced(idx->type())) {
```

Comment:
Why is this a static method call right now? Can we make this a virtual function on the index object?
Or do we need the same function elsewhere in static context too?

Reply:
For now this function only checks index type so I want to avoid virtual call.